themes/bootstrap3/templates/record/checkbox.phtml

```diff
@@ -1,5 +1,5 @@
 <label class="record-checkbox hidden-print">
-  <input class="checkbox-select-item" type="checkbox" name="ids[]" value="<?=$this->escapeHtmlAttr($this->id) ?>"<?php if (isset($this->formAttr)): ?> form="<?=$this->formAttr ?>"<?php endif; ?> aria-label="<?=$this->transEscAttr('select_item')?>">
+  <input <?php if (isset($this->checkboxElementId)): ?>id="<?=$this->checkboxElementId ?>"<?php endif; ?> class="checkbox-select-item" type="checkbox" name="ids[]" value="<?=$this->escapeHtmlAttr($this->id) ?>"<?php if (isset($this->formAttr)): ?> form="<?=$this->formAttr ?>"<?php endif; ?> aria-label="<?=$this->transEscAttr('select_item')?>">
```

Comment:
I think we need to add `$this->escapeHtmlAttr()` around `$this->checkboxElementId` to ensure safe escaping of the value.

Reply:
I wasn't sure whether it was escaped before, but seems to be needed.